### PR TITLE
Renamed QueryRequestOptions -> CosmosQueryRequestOptions

### PR DESCRIPTION
**Pull request by kushagraThapar**
* JUST RENAME
* Renamed QueryRequestOptions -> CosmosQueryRequestOptions

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentQueryExecutionContextFactory.java [hunk 1]
```diff
@@ -127,7 +127,7 @@
                                                                                                               resourceTypeEnum,
                                                                                                               resourceType,
                                                                                                               query,
-                                                                                                              queryRequestOptions,
+                                         cosmosQueryRequestOptions,
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentQueryExecutionContextFactory.java:130]: indentation.

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentQueryExecutionContextFactory.java [hunk 2]
```diff
@@ -200,7 +200,7 @@
                 resourceTypeEnum,
                 resourceType,
                 query,
-                queryRequestOptions,
+            cosmosQueryRequestOptions,
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentQueryExecutionContextFactory.java:203]: indentation.

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/OrderByDocumentProducer.java [hunk 3]
```diff
@@ -77,7 +77,7 @@ private DocumentProducerFeedResponse resultPageFrom(RequestChargeTracker tracker
                 consumeComparer,
                 client,
                 collectionRid,
-                queryRequestOptions,
+            cosmosQueryRequestOptions,
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/OrderByDocumentProducer.java:80]: indentation.

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentQueryExecutionContextFactory.java (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentQueryExecutionContextFactory.java:130]: ah intellij, will fix it here and everywhere else.

**Pull request merged**